chains/evm/scripts/compile_all

```diff
@@ -9,7 +9,7 @@ echo " └───────────────────────
 # The offRamp uses a specific lower optimization runs value. All other contracts use the default value
 # as specified in the foundry.toml.
 OPTIMIZE_RUNS_OFFRAMP=800
-OPTIMIZE_RUNS_FEE_QUOTER=8000
+OPTIMIZE_RUNS_FEE_QUOTER=50
```

Comment:
open to other options here, just reduced it to ensure feeQuoter is under 24.5k byte size limit

Reply:
Is 50 the highest that works or just a random value?

Reply:
4k gives us 24,266byte with 310bytes runtime margin, which should be enough